### PR TITLE
SL-2594 Silence the God Classes Lift comments on linkedin-sdk PRs

### PR DESCRIPTION
**Pull request by dougalrea**
### Description of Changes

`.lift.toml` file added to repo root to silence the God Classes comments from being posted on PRs.

Broken but inessential gh/release step commented out of Circle CI prod deployment workflow

### Documentation

See here: https://community.sonatype.com/t/disabling-god-classes-comment/9379

### Risks & Impacts

Sonatype will still run, but won't post comments on PRs.

### Testing

To be tested by test PR after merge (`.lift.toml` file must be present on dev branch to take effect)

### Compare (For layered PRs)

N/A

## Final Checklist

Please tick once completed.

- [x] Build passes.
- [x] Versioning considered (the version number in this PR is inline with semantic 
versioning requirements).
- [ ] Change log has been updated.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -20,17 +20,17 @@ workflows:
...
-          filters:
-            branches:
-              only: master
-          requires:
-            - predeploy
-          title: Circle CI release
-          tag: v$MVN_VERSION
-          files: target/ebx-linkedin-sdk-${MVN_VERSION}*.jar
-          notes-file: CHANGELOG.md
+      # - gh/release:
```
Comment (jack7ellis) [inline .circleci/config.yml:23]: should like just be removed? we can always retrieve it from git history if we want it back

just to avoid commented out code in the repo

File: .lift.toml [hunk 2]
```diff
@@ -0,0 +1 @@
+disableTools=["refactor-first"]
```
Comment (jack7ellis) [inline .lift.toml:1]: To prove this works, you should revert this change (merge the other change). And create a new PR with this in, to show that the warning isn't made.

Or would that not work, do you have to merge it first? Would be good to verify it does what we want in some way.

**Review by jack7ellis — APPROVED** [review]
A couple of suggestions, but will leave those up to you

File: .circleci/config.yml (hunk 1 above)
Comment (dougalrea) [reply .circleci/config.yml:23]: Sure, happy to remove it. It might even be better to have the gh/release step handled by a GitHub action rather than Circle CI anyway come to think of it...

File: .lift.toml (hunk 2 above)
Comment (dougalrea) [reply .lift.toml:1]: Yeah unfortunately I have to merge it first as I the config file needs to be on dev branch to take effect. I'll open a test PR after merging this one to confirm comment isn't posted, but based on the resolved sonatype-lift comment above, it looks like it will work. The link in PR desc. is very appropriate for our exact situation too, so I'm fairly confident
